README.md

```diff
@@ -4,6 +4,8 @@ Sourcerer hashes your git repos into intelligent engineering profiles.
 Install/uninstall
 =================
 
+Sourcerer requires JRE to be installed on the system. Install last version from [Oracle](http://www.oracle.com/technetwork/java/javase/downloads/jre8-downloads-2133155.html).
+
```

Comment:
as I mentioned already, it wasn't enough in my case, JDK was required. We might want to file a follow up issue to investigate it.